tlv/stream.go

```diff
@@ -183,6 +189,10 @@ func (s *Stream) Decode(r io.Reader) error {
 			return err
 		}
 
+		if length > lnwire.MaxMessagePayload {
```

Comment:
We want to keep the import graph as small as possible for this package, so I would pull this out into its own constant rather than import `lnwire`. It's also okay if it's more conservative, e.g. 100kb